Docker/StorageApi/Writer.php

```diff
@@ -359,6 +380,15 @@ public function uploadTables($source, $configuration = [])
                 The following unset should be removed once the 'escaped_by' parameter is removed from table manifest. */
                 unset($config['escaped_by']);
                 $config["primary_key"] = self::normalizePrimaryKey($config["primary_key"]);
+                if ($config["primary_key"] == [""]) {
```

Comment:
@odinuv that is a strange one,  it's obviously a hack to deal with that `[""]` so I must have encountered that somewhere, but I can't remember.  It probably won't happen anymore sense you guys upgraded the PK handling though.  regardless that `==` makes me cry :)